ddtrace/tracer/option.go

```diff
@@ -536,7 +562,7 @@ func newConfig(opts ...StartOption) *config {
 	// Check if CI Visibility mode is enabled
 	if internal.BoolEnv(constants.CIVisibilityEnabledEnvironmentVariable, false) {
 		c.ciVisibilityEnabled = true               // Enable CI Visibility mode
-		c.httpClientTimeout = time.Second * 45     // Increase timeout up to 45 seconds (same as other tracers in CIVis mode)
+		c.httpClientTimeout = time.Second * 45     // Increase timeout up to 45 seconds (same as o
```

Comment:
That looks like an unintended change?

```suggestion
		c.httpClientTimeout = time.Second * 45     // Increase timeout up to 45 seconds (same as other tracers in CIVis mode)
```

Reply:
Done in https://github.com/DataDog/dd-trace-go/pull/3033/commits/1a458ca46107aeafc2fab5435487ac77d07a5852